layouts/shortcodes/observability_pipelines/processors/remap.en.md

```diff
@@ -47,3 +49,5 @@ To set up the rename field processor:
 
 - Use `outer_key.inner_key` to refer to the key with the value `inner_value`.
 - Use `outer_key.inner_key.double_inner_key` to refer to the key with the value `double_inner_value`.
+
+[10220]: /observability_pipelines/processors/edit_fields
```

Comment:
This link points to Edit fields, not the guide. Is that intentional?

Reply:
It was just wrong 🙃 fixing:

```suggestion
[10220]: /observability_pipelines/guide/remap_reserved_attributes
```